hack/kube/base/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sdps-serviceaccount
+  automountServiceAccountToken: false
```

Comment:
This should be at metadata level, as per https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting